.eslintrc

```diff
@@ -0,0 +1,31 @@
+{
+    "extends": ["prettier", "react-app", "react-app/jest", "airbnb", "import"],
+    "rules": {
+        "comma-dangle": "off",
+        "indent": ["error", 4],
+        "react/jsx-indent": ["error", 4],
+        "react/jsx-indent-props": ["error", 4],
+        "import/prefer-default-export": "off",
+        "import/no-default-export": "error",
+        "react/react-in-jsx-scope": "off",
+        "react/jsx-uses-react": "off",
+        "import/no-cycle": "off",
```

Comment:
add this:
` 'import/no-cycle': [
      'warn',
      {
        ignoreExternal: true,
      },
    ],
`

this rule is very usefull, but it can kinda broke and start checking the node_modules and external packages, with that snippet above it would only check the project source files